src/sentry/search/events/builder/base.py

```diff
@@ -1483,7 +1483,7 @@ def handle_invalid_float(cls, value: float | None) -> float | None:
             return None
         return value
 
-    def run_query(self, referrer: str, use_cache: bool = False) -> Any:
+    def run_query(self, referrer: str | None, use_cache: bool = False) -> Any:
```

Comment:
This should be a no-op 1487 was already handling the cases where referrer is None